README.md

```diff
@@ -105,7 +119,7 @@ command -v nvm
 
 which should output `nvm` if the installation was successful. Please note that `which nvm` will not work, since `nvm` is a sourced shell function, not an executable binary.
 
-### Important Notes
+### Important notes
```

Comment:
typically, section titles in prose are `Title Cased`, not `Sentence cased` (hence, "title case")

Reply:
I agree - I took a swing though seeing as it was the only one which didn't match the precedence set by the others.

Will update